micro_ros_agent/CMakeLists.txt

```
@@ -17,6 +17,11 @@ cmake_minimum_required(VERSION 3.5)
 option(UROSAGENT_GENERATE_PROFILE
   "Generates agent.refs according to the .msgs provided in the .repos" OFF
   )
+
+option(UGENERATE_GRAPH_MANAGER
```

```suggestion
option(MICROROS_AGENT_ENABLE_GRAPH_MANAGER
```

@jamoralp opinions on naming

---

looks right, although a bit long but we would probably generate more confusion by naming it UROS_AGENT_***